### PR TITLE
Limitations: update 'update command' section

### LINKED ISSUE #248: Limitations: update 'update command' section
Only Block IO is not supported by update command, we should update the limitations

### PR DESCRIPTION
**Pull request by devimc**
update 'update command' section

fixes #248

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: Limitations.md [hunk 1]
```diff
@@ -123,15 +123,8 @@ See issue https://github.com/kata-containers/runtime/issues/129 for more informa
...
-The runtime does not currently implement the update command, hence
-does not support some of the `docker update` functionality. Much of the
-`update` functionality is based around cgroup configurations.
-
-It might be possible to implement some of the update functionality by adjusting cgroups either around the VM or inside the container VM, or by some other VM functional equivalent. See [the constraints challenge](#the-constraints-challenge) section for further information on how to handle constraints.
-
-Note that the OCI standard does not specify an `update` command.
-
-See issue https://github.com/kata-containers/runtime/issues/189 for more information.
+Currently only block IO (relative weight) is not supported.
```
Comment (klynnrif) [inline Limitations.md]: Suggested rewrite: Currently, only block I/O  (i.e. relative weight) is not supported.

Comment (devimc) [reply Limitations.md]: Thanks, I'll changes this to `Currently, only block I/O weight is not supported.`

see https://docs.docker.com/engine/reference/run/#runtime-constraints-on-resources

**Comment by devimc** [comment]
@klynnrif  change applied, thanks

**Comment by caoruidong** [comment]
lgtm

**Comment by jodh-intel** [comment]
\o/

lgtm

**Pull request merged**